srcpkgs/doomretro/template

```diff
@@ -9,5 +9,6 @@ short_desc="Refined DOOM source port"
 maintainer="Benjamín Albiñana <benalb@gmail.com>"
 license="GPL-3.0-or-later"
 homepage="https://www.doomretro.com"
+changelog="https://github.com/bradharding/doomretro/releases"
```

Comment:
I believe Chocimier intended for you to use https://raw.githubusercontent.com/bradharding/doomretro/master/releasenotes.md instead.